hackathon/urls.py

```diff
@@ -1,7 +1,8 @@
 from django.urls import path
 
-from .views import HackathonListView
+from .views import HackathonListView, create_event
 
 urlpatterns = [
-    path('', HackathonListView.as_view(), name="hackathon-list")
+    path('', HackathonListView.as_view(), name="hackathon-list"),
+    path("create-event", create_event, name="create_event")
```

Comment:
Please try not to mix double and single quotes. Also, you are mixing names with underscore and hyphen, I would keep that consistent as well. Commonly for URLs an underscore would be used.

Reply:
Also, I would keep the naming consistent and make it `create_hackathon` instead of `create-event`

Reply:
The first line (hackathon-list) was pulled in from a previous PR, so I'll leave that as it is, as I'm not sure of all the places where it is used.

The mixing of underscores and hyphens is my employer's preference for naming conventions - URLs should be hyphenated, whereas views and names should use underscores - so this is another force of habit. If the preference here is to use underscores for all then that's cool. I've change the `create-event` URL, and view and name to `create_hackathon` instead. I've also changed all quotes in the file to single quotes. These changes will be visible when the new changes are pushed to the existing repo.

Reply:
Cool. Thanks!